src/sql/parts/dashboard/widgets/explorer/explorerWidget.component.ts

```diff
@@ -94,7 +95,7 @@ export class ExplorerWidget extends DashboardWidget implements IDashboardWidget,
 			dataSource: this._treeDataSource,
 			filter: this._treeFilter,
 			renderer: this._treeRenderer
-		});
+		}, { horizontalScrollMode: ScrollbarVisibility.Visible });
```

Comment:
This should probably be auto, since we don't want a scrollbar if there is nothing to scroll.

Reply:
yes, the scroll definitely should only be visible when the content overflows the visible area.